FEATURE_GATES.md

```diff
@@ -74,7 +75,6 @@ Features that reach GA and over time become stable will be removed from this tab
 | FallbackConfiguration                   | `false` | Alpha | 3.2.0  | TBD   |
 | KongCustomEntity                        | `false` | Alpha | 3.2.0  | 3.3.0 |
 | KongCustomEntity                        | `true`  | Beta  | 3.3.0  | 3.4.0 |
-| CombinedServicesFromDifferentHTTPRoutes | `false` | Alpha | 3.4.0  | TBD   |
```

Comment:
Instead of removing it from this list let's mark that it has been in alpha in 3.4 until 3.5 (just like for other feature gates):

```diff
| CombinedServicesFromDifferentHTTPRoutes | `false` | Alpha | 3.4.0  | 3.5.0 |
```